tools/e2etesting/GetSecrets.ps1

```diff
@@ -1,8 +1,20 @@
-param($KeyVaultName)
+param(
+	[Parameter(Mandatory=$true)]
```

Comment:
@MaidaB, can you please use 4 spaces for indentation instead of tab character ? The indentation of this file is inconsistent at the moment. I can see tabs being used on lines `47`, `48` and `51` as well.

Reply:
fixed